pkg/scheduler/apis/config/v1beta2/conversion.go

```diff
@@ -49,9 +49,20 @@ func Convert_v1beta2_KubeSchedulerConfiguration_To_config_KubeSchedulerConfigura
 	if err := autoConvert_v1beta2_KubeSchedulerConfiguration_To_config_KubeSchedulerConfiguration(in, out, s); err != nil {
 		return err
 	}
+	convertToInternalProfilePercentageOfNodesToScore(out)
```

Comment:
Could you help me find out where this is hit and UTed?

(In https://github.com/kubernetes/kubernetes/blob/9c9b29032f76e39b0e36200c88fe2f127389a9fe/cmd/kube-scheduler/app/options/options_test.go#L362, this logic is hit, and that test doesn't verify the correctness of the unversioned obj.)

Reply:
@alculquicondor can you shed some light here? Is that we miss a `conversion_test.go`?

Reply:
`scheme_test.go` covers most of it.

Reply:
Added test cases of default and non-default percentages for v1beta2, v1beta3 and v1 in `scheme_test.go`.